digital-objects/ctann/azimuth/v1.0/raw/metadata.yaml

```diff
@@ -2,7 +2,7 @@ title: Crosswalk of Azimuth annotated cells to Cell Ontology
 description: >-
   This crosswalk maps the cell type annotations from Azimuth https://azimuth.hubmapconsortium.org/ to the latest revision of Cell Ontology Version IRI: http://purl.obolibrary.org/obo/cl/releases/2024-05-15/cl.owl in which many new cell types have been added since the original cell type annotations were mapped.
```

Comment:
Provide date for when original mapping was performed? Not very useful info otherwise. Glad to see the 5/15 CL version.

Reply:
The first 7 organs were mapped, but the rest were never mapped to CL or posted to website.  The actual DATES the first 7 were posted are not revealed on the website at all.  I'd have to dig to see if I can obtain the info from the github repo.

Reply:
done